CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ v0.22 + 1
   via `git_config_parse_path()` and `git_config_get_path()`
   respectively.
 
+* `git_index_add_frombuffer()` can now create a blob from memory
+   buffer and add it to the index, even if the index is also in
+   memory.
+
```

Comment:
This does not sound accurate. If the index in purely in-memory, we have no associated repository and thus cannot know where to create the blob (there is even a check for this in the function).